src/components/MainHeader.vue

```diff
@@ -77,28 +75,28 @@ export default {
           "Student Awards": "http://www.cs.cmu.edu/~scsfacts/studentawards.html",
```

Comment:
Also, another bug I discovered, is that links in the menu that are absolute and not relative, leads to a broken URL rather then redirecting to the correct absolute URL. Previously, these were `<a>` tags in the HTML and not `<router-links>`. So, I think you need to add some extra logic (or delete these links for now since these type of legacy links will be handled differently entirely soon) to account for these absolute URLs.

Reply:
Fixed this

Reply:
👍